models/git/branch.go

```diff
@@ -391,7 +391,7 @@ func FindRecentlyPushedNewBranches(ctx context.Context, repoID, userID int64) (B
 			"issue.is_closed":           false,
 		})
 	err := db.GetEngine(ctx).
-		Where("pusher_id=? AND is_deleted=?", userID, false).
+		Where("repo_id=? AND pusher_id=? AND is_deleted=?", repoID, userID, false).
```

Comment:
The problem I see with this is the following:
Now, you won't be able to see this message in forks.
I've thought about the logic in the previous PR and thought it seems sensible.
While I can also see your problem, the question is now which of the two is better?
Or is there a third approach to still include forks, i.e. additionally storing a `ForkedFromID` or something like that?

Reply:
I think it's difficult to get all forked repositories. Maybe only the first level forked?

Reply:
I meant the other way round as it is a `one-to-n` relation: Each fork only has a single parent, so we can check for `repoID = fork or parent`.